src/AcmProgressTracker/AcmProgressTracker.tsx

```diff
@@ -0,0 +1,183 @@
+    container: {
+        display: 'flex',
+    },
+    icon: {
+        width: '18px', // Progress size md is 18px
+    },
+    iconMargin: {
+        margin: '3px 2px 1px 2px',
+    },
+    button: {
+        padding: 0,
+        fontSize: 'inherit',
+    },
+})
+
+const divider = (isStacked: boolean) => {
+    const classes = useStyles()
+    if (isStacked) return undefined
+    return (
+        <svg className={classes.divider}>
+            <line
+                x1="0"
+                x2="100"
+                y1="10"
+                y2="10"
+                stroke="#D2D2D2" // --pf-global--palette--black-300
+                strokeWidth="2"
+                strokeLinecap="square"
+                strokeDasharray="1, 3"
+            />
+        </svg>
+    )
+}
+
+export function AcmProgressTracker(props: AcmProgressTrackerProps) {
+    const classes = useStyles()
+    const { viewWidth } = useViewport()
+    const isStacked = props.isStacked || viewWidth < 700
+
+    return (
+        <div>
+            <TextContent>
+                <Text component="h3">{props.Title}</Text>
+                <Text component="small">{props.Subtitle}</Text>
+            </TextContent>
+            <div>
+                <div className={isStacked ? classes.popoverParentContainer : classes.parentContainer}>
+                    {props.steps.map((step, index) => (
+                        <div key={index} className={classes.stepContainer}>
+                            <div>
+                                <InlineStatus type={step.statusType} status={step.statusText} popover={step.popover} />
+                                <TextContent>
+                                    <Text className={classes.stepStatus} component="small">
+                                        {step.statusSubtitle}
+                                    </Text>
+                                </TextContent>
+                            </div>
+                            {index < props.steps.length - 1 && divider(isStacked)}
+                        </div>
+                    ))}
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export function InlineStatus(props: { type: StatusType; status: string | React.ReactNode; popover?: PopoverProps }) {
+    const classes = useStyles()
+    return (
+        <div className={classes.container}>
+            <div className={classes.icon}>
+                <StatusIcon type={props.type} />
+            </div>
+            <span style={{ marginLeft: '.4rem' }}>
+                {props.popover ? (
+                    <Popover hasAutoWidth {...props.popover}>
+                        <Button variant="link" className={classes.button}>
+                            {props.status}
+                        </Button>
+                    </Popover>
+                ) : (
+                    props.status
+                )}
+            </span>
+        </div>
+    )
+}
+
+function StatusIcon(props: { type: StatusType }) {
```

Comment:
Do we need this copy?

Reply:
AcmInlineStatus uses statusIcon...I moved it over and renamed it for the same reason I moved and re-named Inlinestatus.

Reply:
Removed the copied code